web/src/api/types.ts

```diff
@@ -2,3 +2,10 @@ export type AuthRequest = {
 	name: string
 	password: string
 }
+
+export type UserInfo = {
```

Comment:
Why do you need to define this type?

Reply:
We should get User type for navbar name.